lib/topological_inventory/ansible_tower/parser.rb

```diff
@@ -8,11 +8,23 @@ class Parser < TopologicalInventoryIngressApiClient::Collector::Parser
     include TopologicalInventory::AnsibleTower::Parser::ServicePlan
     include TopologicalInventory::AnsibleTower::Parser::ServiceOffering
 
+    def initialize(tower_host:)
+      super()
+
+      host_url = tower_host
+      host_url = "https://#{host_url}" unless host_url =~ %r{\Ahttps?:\/\/}
```

Comment:
I think this would be a little cleaner with something like `uri = URI(tower_host); uri.scheme ||= "https"; host_url = uri.to_s` but this should do for now